### PR TITLE
Enable acceptance tests for CentOS 6

### PR DESCRIPTION
**Pull request by traylenator**
#### Pull Request (PR) description
Enable acceptance tests for CentOS 6 and Ubuntu 18.04

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
       "operatingsystemrelease": [
         "12.04",
         "14.04",
-        "16.04"
+        "16.04",
+	"18.04"
```
Comment (ekohl) [inline metadata.json]: Odd indenting here due to mixing of tabs and spaces

Comment (traylenator) [reply metadata.json]: Looks better now.

**Comment by ekohl** [comment]
Looks like the module also relies on LSB facts being present.

**Comment by traylenator** [comment]
Okay. I think this is worth another check now.

**Comment by traylenator** [comment]
@ekohl I think all comments addressed

**Pull request merged**